### ISSUE TITLE
The term 'pipx' is not recognized as the name of a cmdlet, function, script file, or operable program

### ISSUE THREAD

**Issue by IshaanShettigar** [smartcontractkit/full-blockchain-solidity-course-py#1197]
I have tried all the methods used in this to install brownie but none seem to work
https://eth-brownie.readthedocs.io/en/stable/install.html
If i use pipx then i get the below message

PS C:\Users\Ishaan\VS code projects\demos\brownie_simple_storage> pipx
pipx : The term 'pipx' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if 
At line:1 char:1
+ pipx
+ 
    + CategoryInfo          : ObjectNotFound: (pipx:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Then if i try 
`pip install eth-brownie`

it works

```
git clone https://github.com/eth-brownie/brownie.git
cd brownie
```

The above lines also work
but when i try to `python3 setup.py install`
It gives me this error

PS C:\Users\Ishaan\VS code projects\demos\brownie_simple_storage\brownie> python3 setup.py install
>>
running install
error: can't create or remove files in install directory

The following error occurred while trying to add or remove files in the
installation directory:

    [Errno 13] Permission denied: 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.9_3.9.2800.0_x64__qbz5n2kfra8p0\\Lib\\site-packages\\test-easy-install-15808.write-test'

The installation directory you specified (via --install-dir, --prefix, or
the distutils default setting) was:

    C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2800.0_x64__qbz5n2kfra8p0\Lib\site-packages\

Perhaps your account does not have write access to this directory?  If the
installation directory is a system-owned directory, you may need to sign in
as the administrator or "root" account.  If you do not have administrative
access to this machine, you may wish to choose a different installation
directory, preferably one that is listed in your PYTHONPATH environment
variable.

For information on other options, you may wish to consult the
documentation at:

  https://setuptools.readthedocs.io/en/latest/deprecated/easy_install.html

Please make the appropriate changes for your system and try again.

Please tell me what to do and how to fix this, I have been stuck on this for 2 hours

**Comment by cromewar** [comment]
Hello @IshaanShettigar 
Just don't use pipx use regular pip, also vscode will be happier if you do so as pipx has import problems.
Also please go to: [chronological-issues-from-video.md](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/blob/main/chronological-issues-from-video.md#python-related-issues) as you are most likely going to get the cytools error later.

**Comment by tayyipcanbay** [comment]
try to run this command before installition:
Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy Unrestricted
I got the an error about permission and this worked for me

**Comment by eiyen** [comment]
Maybe it's because `pipx` is not the environment variable.

If it's true, you need to add the path of pipx to the environment variable manually, after executing the statement `python -m pipx ensurepath` on windows. At least, this method is worked for me.

For more information, maybe [this answer on StackOverflow](https://stackoverflow.com/a/70828216/14640362) will help.

**Comment by tayyipcanbay** [comment]
> Does @tayyipcanbay solution work for you @IshaanShettigar ?

Yes it did. I read something about powershell. I dont remember details but as i remember it was about an authorization problem
